.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -237,7 +237,7 @@ AddType text/vtt                            vtt
     # NOTE THAT THESE ARE A 302 REDIRECT, not a 301. Replace with 301 at CNV GA
     # send all cnv/ to virt/about-virt.html.
     # send all virt/ to virt/about-virt.html excluding about-virt.html otherwise there is an infinite loop
-    # RewriteRule container-platform/4\.9/virt/(?!about-virt\.html) /container-platform/4.9/virt/about-virt.html [NE,R=302]
+    RewriteRule container-platform/4\.10/virt/(?!about-virt\.html) /container-platform/4.10/virt/about-virt.html [NE,R=302]
```

Comment:
@sjhala-ccs we have tested this redirect on a couple of different htaccess rewrite test programs, with the same result. I provided the URL https://docs.openshift.com/container-platform/4.10/virt/virt-learn-more-about-openshift-virtualization.html and the following rule:

`RewriteRule container-platform/4\.10/virt/(?!about-virt\.html) /container-platform/4.10/virt/about-virt.html [NE,R=302]`

The URL that is output after being processed by the rule is not the required one:

https://docs.openshift.com/container-platform/4.10/virt/about-virt.htmlvirt-learn-more-about-openshift-virtualization.html

It should be just https://docs.openshift.com/container-platform/4.10/virt/about-virt.html.

We are reviewing how to meet the requirement and will update as soon as possible.

Reply:
Thanks @pneedle-rh. I have updated the PR. I am still confused as to why the rule worked fine for previous releases. Do I have your okay to merge this one?